[bugfix] Remove Makefile

trombik opened the pull request:
instead of using Makefile, use `changed` attribute in the task that creates tables.

fixes #23

Commit:
remove Makefile, run postmap on changed tables

Commit:
make is not required anymore

Commit:
remove unused var

Commit:
also remove make from empty_aliases_spec.rb

mitsururike commented on tasks/main.yml:
```diff
@@ -189,15 +162,16 @@
+    - item.item.name in postfix_text_databases
+    # for a reason I do not know, "not (var + 'string' | exists)" does not work
+    # here
+    - "item.changed or not [item.invocation.module_args.dest, 'db'] | join('.') | exists"
```
"item.changed or not ( [item.invocation.module_args.dest, 'db'] | join('.') | exists )"
using parentheses in a long expression helps to read and reduces mistakes.

trombik replied:
fixed in ede2e56

Commit:
remove Makefle tests from empty_aliases_spec.rb

Commit:
use parenthesis for readability

https://github.com/reallyenglish/ansible-role-postfix/pull/28#discussion_r119010226
